Header header

string name
float32 R3_x
float32 R3_y
float32 L3_x
float32 L3_y
float32 X
float32 T
float32 O
float32 Sq
float32 L1
float32 R1